drone/src/agent/engine.rs

```diff
@@ -16,13 +16,13 @@ pub enum EngineBackendStatus {
     Running { addr: SocketAddr },
 
     /// The backend exited on its own without indicating failure.
-    Finished,
+    Exited,
 
     /// The backend exited on its own with a failure state.
     Failed,
 
     /// The backend was terminated by external forces.
-    Terminated,
+    Swept,
```

Comment:
Why this change? To me, "Terminated" feels like the right terminology for the engine level, while "Swept" feels more Plane-specific.

Reply:
I agree, it was an attempt at consistency but now that you mention it, I think it's more confusing to use “swept” here.

Reply:
+1 for terminated